### DISCUSSION TITLE
Font size, how to change?

### DISCUSSION THREAD

**Discussion by HubKing** [hoppscotch/hoppscotch#3749] [Q&A] [answered]
[The doc says I can change it in the settings](https://docs.hoppscotch.io/documentation/features/customization),
![image](https://github.com/hoppscotch/hoppscotch/assets/10900489/cbe2e4fe-e168-494a-ab53-15291ea6da93)

but I cannot find it in the settings.
![image](https://github.com/hoppscotch/hoppscotch/assets/10900489/cdebc7e0-220b-40c9-bd61-28ea5144837e)

Also, ctrl + plus does not work.

Hoppscotch 23.12.1-1 from AUR
![image](https://github.com/hoppscotch/hoppscotch/assets/10900489/66ff1d5c-379d-46c2-b140-7c13eafef373)

**Comment by liyasthomas** [comment, marked as answer]
The ability to change font size has been deprecated in favor of the browser's native zoom +/- capabilities. This feature is yet to be released on desktop app.

**Comment by supersonictw** [comment]
Tracing:
https://github.com/hoppscotch/hoppscotch/issues/3545

**Comment by arifur-rahman-arif** [comment]
We need the zoom feature for desktop app. Using Fedora linux

**Comment by tangkunyin** [comment]
Need the zoom feature for desktop app too, using macOS.

[The problem](https://github.com/hoppscotch/hoppscotch/issues/4035) have been remained to 2025 already, the latest desktop app `Version 24.12.0-0 (20241223.142737)` 🙈

Many many many thanks 🙏🏻

**Comment by Pshemas** [comment]
+1 on this, without it the text is simply too small on hidpi displays. Or at least enable zoom-in, so I could increase font size by ctrl + / ctrl - .

**Comment by woeoio** [comment]
Depending on the zoom function of the browser I think is affecting the user experience, highly recommend the font size setting option

**Comment by woeoio** [comment]
Also, when the browser zooms in, it becomes blurry

**Comment by woeoio** [comment]
For example, the font of a product I am currently using is more coordinated, please refer to it
I hope hoppscotch will surpass these products in the future, and I am ready to migrate here
![apifox-ui](https://github.com/user-attachments/assets/b53ed76c-22f2-48fc-a5d0-a235d5efe5b8)

**Comment by zeevenn** [comment]
+1 need custom font size, it's too small in desktop.

**Comment by mcwerki2** [comment]
+1 same to little font size for me on desktop

**Comment by Immask-rgb** [comment]
+1 need custom font size, it's too small in desktop.

**Comment by Clyde256** [comment]
I would also appreciate the font size increase on the desktop, it is really small.

**Comment by cimpok** [comment]
+1 need custom font size, it's too small in desktop.

**Comment by th3n3rd** [comment]
This is very frustrating.

That said, for anyone that is using it on the desktop (Mac OS at least): 

1. Right click anywhere on the app
2. Click on `Inspect Element`
3. Select the `<html ..>` tag
4. Look on the CSS panel and update the `--font-size-body` property to something more than `0.75rem`, I am using `1.2rem`

Hope this helps.
